app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-related.scss

```diff
@@ -15,6 +15,22 @@
   padding: 0;
 }
 
+.gem-c-step-nav-related-singular {
```

Comment:
This class name doesn't follow our [BEM convention](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#styles) - should be `gem-c-step-nav-related--singular`.